src/app/settings/settings.component.html

```diff
@@ -0,0 +1,169 @@
+<ion-header class="ion-no-border ion-no-shadow">
+	<ion-toolbar>
+		<ion-buttons slot="start">
+			<ion-back-button></ion-back-button>
+		</ion-buttons>
+		<ion-buttons slot="end">
+			<ion-button data-testid="c-networks__button--menu">
+				<ion-icon
+					class="text-base"
+					color="dark"
+					src="/assets/svg/menu.svg"
+				></ion-icon>
+			</ion-button>
+		</ion-buttons>
```

Comment:
There must be an ion-title here, it will be displayed when scrolling.
```suggestion
		</ion-buttons>
		<ion-title>{{ "SETTINGS_PAGE.SETTINGS" | translate }}</ion-title>
```